clients/lsp-lua.el

```diff
@@ -592,6 +592,73 @@ and `../lib` ,exclude `../lib/temp`.
   :priority -3
   :server-id 'lsp-lua-lsp))
 
+;;; lua-roblox-language-server
+(defgroup lsp-roblox-lua-language-server nil
+  "Roblox Lua LSP client, provided by the Roblox Lua Language Server."
+  :group 'lsp-mode
+  :version "7.1"
+  :link '(url-link "https://github.com/NightrainsRbx/RobloxLsp"))
+
+(defcustom lsp-clients-roblox-lua-language-server-install-dir (f-join lsp-server-install-dir "roblox-lua-language-server/")
+  "Installation directory for Lua Language Server."
+  :group 'lsp-roblox-lua-language-server
+  :version "7.1"
+  :risky t
+  :type 'directory)
+
+(defcustom lsp-clients-roblox-lua-language-server-bin
+  (f-join lsp-clients-roblox-lua-language-server-install-dir
+          "extension/server/bin/"
+          (pcase system-type
+            ('gnu/linux "Linux/lua-language-server")
+            ('darwin "macOS/lua-language-server")
+            ('windows-nt "Windows/lua-language-server.exe")
+            (_ "Linux/lua-language-server")))
+  "Location of Lua Language Server."
+  :group 'lsp-roblox-lua-language-server
+  :version "7.1"
+  :risky t
+  :type 'fil)
+
+(defcustom lsp-clients-roblox-lua-language-server-main-location
+  (f-join lsp-clients-roblox-lua-language-server-install-dir
+          "extension/server/main.lua")
+  "Location of Roblox Lua Language Server main.lua."
+  :group 'lsp-roblox-lua-language-server
+  :version "7.1"
+  :risky t
+  :type 'file)
+
+(defun lsp-clients-roblox-lua-language-server-test ()
+  "Test Lua language server binaries and files."
+  (and (f-exists? lsp-clients-roblox-lua-language-server-main-location)
+       (f-exists? lsp-clients-roblox-lua-language-server-bin)))
+
+(defun lsp-roblox-lua-language-server-install (client callback error-callback update?)
+  "Download the latest version of roblox-lua-language-server and extract it to
+`lsp-roblox-lua-language-server-install-dir'."
+  (ignore client update?)
```

Comment:
You can add `_` prefix to `_client` `_update` for it to be ignored instead of this